src/course-outline/CourseOutline.jsx

```diff
@@ -107,12 +108,15 @@ const CourseOutline = ({ courseId }) => {
                       openEnableHighlightsModal={openEnableHighlightsModal}
                     />
                     <div className="pt-4">
+                      {/* TODO add create new section handler in EmptyPlaceholder */}
                       {sectionsList.length ? sectionsList.map((section) => (
                         <SectionCard
                           section={section}
                           onOpenHighlightsModal={handleOpenHighlightsModal}
                         />
-                      )) : null}
+                      )) : (
+                        <EmptyPlaceholder onCreateNewSection={null} />
```

Comment:
it's better to use here `onCreateNewSection={() => ({})}` if you have `onCreateNewSection` is required either you have and props error.

Reply:
Done